src/libaktualizr/utilities/utils.h

```diff
@@ -44,7 +44,7 @@ struct Utils {
   static void createDirectories(const boost::filesystem::path &path, mode_t mode);
   static bool createSecureDirectory(const boost::filesystem::path &path);
   static std::string urlEncode(const std::string &input);
-  static CURL *curlDupHandleWrapper(CURL *curl_in, bool using_pkcs11);
+  static CURL *curlDupHandleWrapper(CURL *curl_in, bool using_pkcs11, CURLSH *share);
```

Comment:
Maybe we don't need this workaround anymore at all. If so, then in the case of of single curl handle, I suppose there is no need to use the share object at all since libcurl is supposed to apply some caching to the single handle.

Reply:
```
  // This is a workaround for a bug in curl. It has been fixed in
  // 75a845d8cfa71688d59d43788c35829b25b6d6af (curl 7.61.1), but that is not
  // the default in most distributions yet, so we will continue to use the
  // workaround.
  if (using_pkcs11) {
```

The LmP's current curl version is `7.82.0`, and aklite goes together with LmP within the same version, so I supposed we can get rid of this workaround.